Sources/StreamChatUI/ChatChannelList/ChatChannelUnreadCountView+SwiftUI.swift

```diff
@@ -7,23 +7,23 @@ import SwiftUI
 
 @available(iOS 13.0, *)
 /// Protocol of `_ChatChannelUnreadCountView` wrapper for use in SwiftUI.
-public protocol _ChatChannelUnreadCountViewSwiftUIView: View {
-    associatedtype ExtraData: ExtraDataTypes
-    init(dataSource: _ChatChannelUnreadCountView<ExtraData>.ObservedObject<Self>)
+public protocol ChatChannelUnreadCountViewSwiftUIView: View {
+//    associatedtype ExtraData: ExtraDataTypes
```

Comment:
Why not delete this?

Reply:
It actually should not be comented! I was trying something and forgot about it. Thank you for noticing! 🙌🏻